src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -282,28 +299,33 @@
       [{{collectible-id :id :as collectible} :collectible
         aspect-ratio                         :aspect-ratio
         gradient-color                       :gradient-color}]]
+   {:db (assoc-in db
+         [:wallet :ui :collectible]
+         {:details        collectible
+          :aspect-ratio   aspect-ratio
+          :gradient-color gradient-color})
+    :fx [[:dispatch [:wallet/get-collectibles-by-unique-id-async collectible-id]]
+         ;; We delay the navigation because we need re-frame to update the DB on time.
+         ;; By doing it, we skip a blink while visiting the collectible detail page.
+         [:dispatch-later
+          {:ms       17
```

Comment:
This value looks odd. Is it arbitrary, or was it calculated?
Since `:dispatch-later` relies on JavaScript's `setTimeout` to schedule the call to `dispatch`, and `setTimeout` is not precise, should we round this value?

What do you think?

Reply:
I didn’t write this part and I’m not aware of the reasoning behind choosing this number. As for rounding it, I think it could be a good idea if setTimeout’s precision is a concern.

Reply:
I think we should round it to 20 ms, we are using this value for similar purposes on other flows.

Reply:
Done